### PR TITLE
Fixed : Rust version requirement is not mentioned in the developer's guide.

### LINKED ISSUE #1894: Rust version requirement is not mentioned in the developer's guide.
**Describe the bug**
In the developer's guide, there is no mention of which version of Rust is required. 

**Expected behavior**
We should add something like
```
rustup update stable
```
in the `Bootstrap environment` part, because we always use the latest stable version of Rust to test this project.

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey @HaoYang670 I have added rustup update stable in in the Bootstrap environment part, because we always use the latest stable version of Rust to test this project.
Please review this. If any changes needed inform me i will do that.

Thanks!
#1894 
closes #1894

### CODE REVIEW

**Comment by HaoYang670** [comment]
@alamb please trigger the workflows, thank you!

**Comment by HaoYang670** [comment]
Maybe you could add some context. @NaincyKumariKnoldus

**Comment by NaincyKumariKnoldus** [comment]
> Maybe you could add some context. @NaincyKumariKnoldus

context such as? can you suggest I will add that.

File: DEVELOPERS.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ DataFusion is written in Rust and it uses a standard rust toolkit:
 
 Testing setup:
```
Comment (alamb) [inline DEVELOPERS.md:36]: I wonder if we should add a short note saying "DataFusion uses the latest stable release of rust" as well?

**Review by alamb — APPROVED** [review]
Thanks @NaincyKumariKnoldus !

File: DEVELOPERS.md (hunk 1 above)
Comment (NaincyKumariKnoldus) [reply DEVELOPERS.md:36]: yes @alamb and @HaoYang670 I have added now you can check.

**Review by alamb — APPROVED** [review]

**Comment by alamb** [comment]
Thank you @NaincyKumariKnoldus

**Pull request merged**